### PR TITLE
[Testing] WorkshopOptimizerPlugin 0.1.7.0

### PR DESCRIPTION
**Pull request by belzaru17**
Workshop Optimizer tries to help you manage what to craft in your island sactuary workshops.  It can gather the current supply/demand information by clicking a button, and either exportit to CSV or using it internally.

It looks at combinatins of items and estimated sell value and makes recommendations on what to craft.

### CODE REVIEW

**Comment by philpax** [comment]
This plugin is now in the plugin review queue; we hope to get back to you soon!

**Comment by reiichi001** [comment]
bleatbot, approve

File: testing/live/WorkshopOptimizerPlugin/manifest.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [plugin]
 repository = "https://github.com/belzaru17/WorkshopOptimizerPlugin"
```
Comment (MidoriKami) [inline testing/live/WorkshopOptimizerPlugin/manifest.toml:2]: Change this to:
`repository = "https://github.com/belzaru17/WorkshopOptimizerPlugin.git"`

**Comment by philpax** [comment]
Hi there, to help us review this plugin, could you write up a brief description of what it does and what kind of functionality it offers? See https://github.com/goatcorp/DalamudPluginsD17/pull/461 for a good example 🙂

**Comment by belzaru17** [comment]
> Hi there, to help us review this plugin, could you write up a brief description of what it does and what kind of functionality it offers? See #461 for a good example 🙂

Hi Philpax, thanks for the review!  I updated the description to add some more details and added a few screenshots to make it easier to visualize.

Thanks!

**Review by philpax — APPROVED** [review]

**Comment by philpax** [comment]
Very sorry about how long this has taken! If you're in the goat place Discord, let me know your Discord username and I'll have you added to the private channels.

**Comment by belzaru17** [comment]
No  worries at all, thanks! Discord username Morgand#3813.

**Pull request merged**